src/TfvcMigrator/Program.cs

```diff
@@ -565,6 +574,8 @@ private static void ReportProgress(int changeset, int total, TimedProgress timed
                 var name = gitIdentity[..openingAngleBracketIndex].TrimEnd().ToString();
                 var email = gitIdentity[(openingAngleBracketIndex + 1)..^1].Trim().ToString();
 
+                if (email == "email") throw new NotImplementedException("<email> in authors file must be replaced with a valid email address");
```

Comment:
These NotImplementedExceptions predate the PR, but since the issue design does ask for a friendly error message and exit code of 1, maybe now is the time to implement all of these.

Sometimes I define a CommandLineException class where we'd do `throw new CommandLineException("...", exitCode: 1);` and have a try...catch handle that nicely at the top level. (http://source.techsola.com/#q=CommandLineException)

Reply:
I went ahead and wrapped the body of MigrateAsync in the try/catch; it seemed more appropriate than the main method, but it does make the diff a little harder to read because of the indentation changes. Let me know if you'd prefer it in Main